reconcile/utils/ocm.py

```diff
@@ -931,14 +937,44 @@ def _init_blocked_versions(self, blocked_versions):
                 raise TypeError(f"blocked version is not a valid regex expression: {b}")
 
     @retry(max_attempts=10)
-    def _get_json(self, api):
-        r = requests.get(f"{self.url}{api}", headers=self.headers)
+    def _do_get_request(self, api: str, params: Mapping[str, str]) -> Mapping[str, Any]:
+        r = self.session.get(
+            f"{self.url}{api}",
+            params=params,
+            timeout=REQUEST_TIMEOUT_SEC,
+        )
         r.raise_for_status()
         return r.json()
 
+    @staticmethod
+    def _response_is_list(rs: Mapping[str, Any]) -> bool:
+        return rs["kind"].endswith("List")
+
+    def _get_json(self, api: str) -> Mapping[str, Any]:
```

Comment:
Same as above, `dict` is fine for the return value type.

Reply:
done